bundles/org.openhab.binding.luxtronikheatpump/src/main/resources/OH-INF/thing/channels.xml

```diff
@@ -0,0 +1,2078 @@
+		<item-type>Switch</item-type>
+		<label>Output "Auxiliary Circulation Pump"</label>
+		<state readOnly="true"/>
+	</channel-type>
+
+	<channel-type id="outputControlSignalAdditionalHeating" advanced="true">
+		<item-type>Switch</item-type>
+		<label>Output "Control Signal add. Heating"</label>
+		<state readOnly="true"/>
+	</channel-type>
+
+	<channel-type id="outputFaultSignalAdditionalHeating" advanced="true">
+		<item-type>Switch</item-type>
+		<label>Output "Control Signal add. Heating/Fault Signal"</label>
+		<state readOnly="true"/>
+	</channel-type>
+
+	<channel-type id="outputAuxiliaryHeater3" advanced="true">
+		<item-type>Switch</item-type>
+		<label>Output "Auxiliary Heater 3"</label>
+		<state readOnly="true"/>
+	</channel-type>
+
+	<channel-type id="outputMixingCircuitPump2" advanced="true">
+		<item-type>Switch</item-type>
+		<label>Output "Pump Mixing Circuit 2"</label>
+		<state readOnly="true"/>
+	</channel-type>
+
+	<channel-type id="outputSolarChargePump" advanced="true">
+		<item-type>Switch</item-type>
+		<label>Output "Solar Charge Pump"</label>
+		<state readOnly="true"/>
+	</channel-type>
+
+	<channel-type id="outputSwimmingPoolPump" advanced="true">
+		<item-type>Switch</item-type>
+		<label>Output "Swimming Pool Pump"</label>
+		<state readOnly="true"/>
+	</channel-type>
+
+	<channel-type id="outputMixingCircuit2Closed" advanced="true">
+		<item-type>Switch</item-type>
+		<label>Output "Mixing Circuit 2 Closed"</label>
+		<state readOnly="true"/>
+	</channel-type>
+
+	<channel-type id="outputMixingCircuit2Open" advanced="true">
+		<item-type>Switch</item-type>
+		<label>Output "Mixing Circuit 2 Up"</label>
+		<state readOnly="true"/>
+	</channel-type>
+
+	<channel-type id="operationTimeCompressor1">
+		<item-type>Number:Time</item-type>
+		<label>Operation Time Compressor 1</label>
+		<category>Time</category>
+		<state pattern="%.1f %unit%" readOnly="true"/>
+	</channel-type>
+
+	<channel-type id="pulsesCompressor1">
+		<item-type>Number:Dimensionless</item-type>
+		<label>Pulses Compressor 1</label>
+		<state readOnly="true"/>
+	</channel-type>
+
+	<channel-type id="operationTimeCompressor2" advanced="true">
+		<item-type>Number:Time</item-type>
+		<label>Operation Time Compressor 2</label>
+		<category>Time</category>
+		<state pattern="%.1f %unit%" readOnly="true"/>
+	</channel-type>
+
+	<channel-type id="pulsesCompressor2" advanced="true">
+		<item-type>Number:Dimensionless</item-type>
+		<label>Pulses Compressor 2</label>
+		<state readOnly="true"/>
+	</channel-type>
+
+	<channel-type id="operationTimeSecondHeatGenerator1" advanced="true">
+		<item-type>Number:Time</item-type>
+		<label>Operation Time Second Heat Generator 1</label>
+		<category>Time</category>
+		<state pattern="%.1f %unit%" readOnly="true"/>
+	</channel-type>
+
+	<channel-type id="operationTimeSecondHeatGenerator2" advanced="true">
+		<item-type>Number:Time</item-type>
+		<label>Operation Time Second Heat Generator 2</label>
```

Comment:
```suggestion
		<label>Runtime 2nd Heat Generator 2</label>
```

Reply:
I guess I might need to rename this or the other channel that actually has a similar name. Seems one contains the time it's currently running since and this one contains the total time it has run. guess i will use `runtimeCurrent` and `runtimeTotal` instead of operation time